content/20.treatments.md

```diff
@@ -348,6 +348,18 @@ A quicker treatment regimen with HCQ needs to be investigated to elucidate how t
 Overall, the study provides promising data, although all of the findings still need to be validated in independent population cohorts.
 Their safety analysis indicated that the adverse effects of administering HCQ to mild and moderate COVID-19 cases were manageable.
 Further investigation is necessary to confirm whether the drug indeed ameliorates symptoms and reduces inflammatory response.
+A retrospective analysis [@doi:10.1101/2020.04.16.20065920] was conducted on the data from 368 SARS-CoV-2 patients in all United States Veteran Health Administration medical centers.
+The study investigated the effects of HCQ (n=97), HCQ+AZ (Azithromycin) (n=113) and no HCQ (n=158) in 368 patients retrospectively for primary outcome of death and need for mechanical ventilation. 
+Standard supportive care was rendered to all patients.
+The study included only men of median age 65 years while women were excluded due to considerably less in number (17). 
+Deaths in HCQ only was 27.8%, 22.1% in HCQ+AZ and 14.1% in no HCQ cohort.
+Death risk was higher in HCQ only group compared to no HCQ group (adjusted hazard ratio, 2.61, p<0.03).
+Further, the risk of ventilation was similar in all three groups: (adjusted hazard ratio, 1.43 (HCQ) and 0.43 (HCQ+AZ) compared to HCQ). The study showed evidence of increased mortality on the use of HCQ in this cohort of SARS-CoV-2.
+Also, no advantage was rendered upon the use of HCQ and HCQ+AZ towards mechanical ventilation. 
+The study had few limitations.
+The study was not randomized control and did not include placebo.
+Also, the baseline vital signs, laboratory tests, prescription drug use were significantly different in three groups which can potentially have an effect on treatment outcome. 
+Further the authors acknowledge that the effect of the drugs might be different in females as well as in pediatric subjects as these subjects were not the part of the study.
```

Comment:
Furthermore, the authors acknowledge that the effect of the drugs might be different in females and pediatric subjects, since these subjects were not the part of the study.